### PR TITLE
update libpng pin

### PR DESCRIPTION
**Pull request by ocefpaf**
@isuruf what do you think? I am updating `libpng` now.

https://abi-laboratory.pro/tracker/timeline/libpng/

### CODE REVIEW

**Comment by jakirkham** [comment]
@ccordoba12, would this be compatible with `qt` in `defaults`?

File: scripts/pin_the_slow_way.py [hunk 1]
```diff
@@ -65,7 +65,7 @@
           'libiconv' :'libiconv 1.15',  # 1.15
           'libmatio': 'libmatio 1.5.*',  # NA
           'libnetcdf': 'libnetcdf 4.4.*',  # 4.4.1
-          'libpng': 'libpng >=1.6.22,<1.6.31',  # 1.6.27
+          'libpng': 'libpng >=1.6.33,<1.6.35',  # 1.6.32
```
Comment (isuruf) [inline scripts/pin_the_slow_way.py]: This can also be `>=1.6.32,<1.6.35`

File: scripts/pin_the_slow_way.py [hunk 2]
```diff
@@ -65,11 +65,11 @@
           'libiconv' :'libiconv 1.15',  # 1.15
           'libmatio': 'libmatio 1.5.*',  # NA
           'libnetcdf': 'libnetcdf 4.4.*',  # 4.4.1
-          'libpng': 'libpng >=1.6.22,<1.6.31',  # 1.6.27
+          'libpng': 'libpng >=1.6.32,<1.6.35',  # 1.6.32
           'librdkafka': 'librdkafka 0.9.4|0.9.5',  # NA
           'libssh2': 'libssh2 1.8.*',  # 1.8.0
           'libsvm': 'libsvm 3.21|3.21.*',  # NA
-          'libtiff': 'libtiff >=4.0.3,<4.0.8',  # 4.0.6
+          'libtiff': 'libtiff 4.0.9',  # 4.0.9
```
Comment (ocefpaf) [inline scripts/pin_the_slow_way.py]: How about this one @isuruf ?

https://abi-laboratory.pro/tracker/timeline/libtiff/

Comment (isuruf) [reply scripts/pin_the_slow_way.py]: `>=4.0.8,<4.0.10`

File: scripts/pin_the_slow_way.py [hunk 3]
```diff
@@ -41,12 +41,12 @@
...
           'bzip2': 'bzip2 1.0.*',  # 1.0.6
-          'cairo': 'cairo 1.14.*',  # 1.12.18
+          'cairo': 'cairo 1.14.*',  # 1.14.10
           'curl': 'curl >=7.44.0,<8',  # 7.54.1
           'ffmpeg': 'ffmpeg >=3.2.3,<3.2.6',  # NA
           'flann': 'flann 1.9.1',  # NA
-          'fontconfig': 'fontconfig 2.12.*',  # 2.12.1
-          'freetype': 'freetype 2.7',  # 2.5.5
+          'fontconfig': 'fontconfig 2.12.*',  # 2.12.4
+          'freetype': 'freetype 2.8.1',  # 2.8
```
Comment (ocefpaf) [inline scripts/pin_the_slow_way.py]: And fontconfig @isuruf ? Can you confirm this one?

File: scripts/pin_the_slow_way.py [hunk 4]
```diff
@@ -41,16 +41,16 @@
...
           'gdal': 'gdal 2.2.*',  # 2.2.0
           'geos': 'geos 3.6.2',  # 3.5.0
           'giflib': 'giflib 5.1.*',  # NA
-          'glib': 'glib 2.51.*',  # 2.50.2
+          'glib': 'glib 2.55.*',  # 2.53.6
```
Comment (ocefpaf) [inline scripts/pin_the_slow_way.py]: Sending this version in a moment. See https://abi-laboratory.pro/tracker/timeline/glib/

Comment (scopatz) [reply scripts/pin_the_slow_way.py]: There seem to be two removed symbols between 2.51 and 2.55. It is probably worth updating the pin.

**Comment by ocefpaf** [comment]
@isuruf if you have some time can you take a second look at this one before we merge it?

**Comment by ocefpaf** [comment]
Thanks @isuruf!

**Pull request merged**